tests/js/spec/views/projectsDashboard/index.spec.jsx

```diff
@@ -322,24 +439,29 @@ describe('ProjectsDashboard', function () {
       jest.useRealTimers();
       await tick();
       await tick();
```

Comment:
should be able to remove this, might need to use `await waitFor(() => expect(whatever).toBe())`